src/package_io/sanity_check.star

```diff
@@ -1,6 +1,7 @@
 ROOT_PARAMS = [
     "observability",
     "challengers",
+    "superchains",
     "interop",
```

Comment:
"interop" should be removed now, right?

Reply:
nope it stays here in this PR, no destructive changes yet